features/newts/src/main/java/org/opennms/netmgt/newts/NewtsWriter.java

```diff
@@ -202,12 +201,7 @@ public String toString() {
 
     @Override
     public void onEvent(SampleBatchEvent event) throws Exception {
-        // We'd expect the logs from this thread to be in collectd.log
-        Logging.putPrefix("collectd");
```

Comment:
why are we removing this Logging ?

Reply:
This isn't removing logging, it's just not messing with the prefix and leaving it to whatever it otherwise would be. When it sets the logging prefix to `collectd`, a few bad things were happening:
1. The component here isn't `collectd`, it's the newts writer (or the ring buffer which also does the same thing). The ring buffer is running inside of karaf, so its logs will go there, and I suspect the same for newts.
2. Although some of the data through here might come from `collectd`, not all of it does. Data will also commonly come from `pollerd`, as well as other sources that generate metrics.
3. Some of the logging on errors in the ring buffer/timeseriesstorage related to initialization was confusingly showing up in the collectd log, making problems harder to chase down (especially when I ran headfirst into https://opennms.atlassian.net/browse/NMS-15285 ). Eeek! These two problems made it take awhile before we noticed this: https://opennms.atlassian.net/browse/NMS-15282?focusedCommentId=69239

Reply:
Except for  response time (Pollerd), most of the persisting happens from Collectd. It would be good to separate logs from  karaf.log into their individual component specific logs.  

Also note that Newts may run outside Karaf when we use Newts alone . So not sure where they end up if we remove this prefix.

If we want to break this further, we can create another logging prefix for timeseries/newts but removing prefix may not be preferred solution ( especially for newts as customers may expect logs in collectd.log)

Reply:
Thanks for the context. I removed this change from `NewtsWriter`.

I left the same change in `RingBufferTimeseriesWriter`, but it emits almost no logs in my experience except instantiation, destroy and when there are problems.